packages/forms-web-app/src/views/final-comment/final-comment.njk

```diff
@@ -0,0 +1,72 @@
+{# todo: change layout based on appeal type when component reused #}
+{% extends "layouts/full-appeal-banner/main.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Your final comments - Submit final comments - Appeal a planning decision - GOV.UK" %}
```

Comment:
should this be Submit final comment

Reply:
Updated (also updated the validation error message from "final comments" to "final comment"